### PR TITLE
Bug 1940558: Update Jenkins to 2.277.3 and disable startup wizard

### PR DESCRIPTION
**Pull request by waveywaves**

### CODE REVIEW

File: 2/contrib/openshift/configuration/init.groovy.d/setup_wizard_disable.groovy [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!groovy
+
+import jenkins.model.*
+import hudson.util.*;
+import jenkins.install.*;
+
+def instance = Jenkins.getInstance()
+
+instance.setInstallState(InstallState.INITIAL_SETUP_COMPLETED)
```
Comment (jkhelil) [inline 2/contrib/openshift/configuration/init.groovy.d/setup_wizard_disable.groovy:9]: Is this needed, if we have the system property, is it suffisant ?

Comment (waveywaves) [reply 2/contrib/openshift/configuration/init.groovy.d/setup_wizard_disable.groovy]: System property is sufficient, I tested it.

**Pull request merged**